src/util/irep_hash_container.h

```diff
@@ -9,19 +9,22 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_UTIL_IREP_HASH_CONTAINER_H
 #define CPROVER_UTIL_IREP_HASH_CONTAINER_H
 
+#include <algorithm>
+#include <climits>
```

Comment:
Do you actually need those includes anymore? I'm aware they were necessary in the first version, but now it seems you shouldn't really need them, but I could be wrong.